src/ru/skypro/Main.java

```diff
@@ -1,8 +1,79 @@
 package ru.skypro;
 
-public class Main {
+public class Main<arr> {
+    static Employee[] employee = new Employee[10];
 
     public static void main(String[] args) {
-	// write your code here
+        // write your code here
+
+        employee[0] = new Employee("Иван", "Иванович", "Иванов", 1, 54_200.55);
+        employee[1] = new Employee("Петр", "Иванович", "Петров", 2, 44_985.37);
+        employee[2] = new Employee("Мария", "Семеновна", "Петрова", 3, 47_230.34);
+        employee[3] = new Employee("Семён", "Владимирович", "Смирнов", 4, 68_978.91);
+        employee[4] = new Employee("Екатерина", "Владимировна", "Синица", 4, 67_545.05);
+        employee[5] = new Employee("Владимир", "Витальевич", "Гребнев", 3, 57_496);
+        employee[6] = new Employee("Леонид", "Эдуардович", "Рева", 5, 78_654.66);
+        employee[7] = new Employee("Марина", "Алексеевна", "Савченко", 5, 68_052.78);
+        employee[8] = new Employee("Кристина", "Олеговна", "Вишневская", 2, 46_258.2);
+        employee[9] = new Employee("Алексей", "Александрович", "Васильев", 1, 55_679.43);
+
+
+        System.out.println("Сумма затрат на зарплаты составила " + summa(employee) + " рублей");
+        System.out.println("Максимальная зарплата " + maxSalary(employee) + " рублей");
+        System.out.println("Минимальная зарплата " + minSalary(employee) + " рублей");
+        System.out.println("Средняя зарплата " + midSalary(employee) + " рублей");
+        names(employee);
+        list(employee);
+    }
+
+    static void list(Employee[] employee) {
+        for (Employee a : employee) {
+            System.out.println(a);
+        }
+    }
+
+    static double summa(Employee[] emp) {
+        double sum = 0d;
+        for (Employee num : emp) {
+            sum = sum + num.getSalary();
+        }
+        return sum;
+    }
+
+    static double minSalary(Employee[] emp) {
+        double zp;
```

Comment:
старайтесь придерживаться более полного именования переменных, также касается и других методов
```suggestion
        double minSalary;
```